webapp/app/[locale]/get-started/_components/addChainAutomatically.tsx

```diff
@@ -115,8 +115,8 @@ export const AddChainAutomatically = function ({ chain, layer }: Props) {
       onClick={onClick}
     >
       <div className="flex flex-row gap-x-1">
-        <ChainLogo chainId={chain.id} />
-        <span className="ml-1 text-neutral-950">{chain.name}</span>
+        <div className="w-7"><ChainLogo chainId={chain.id} /></div>
+        <span className="text-neutral-950">{chain.name}</span>
```

Comment:
I think adding the width is enough. We can keep the margin on the `span`. And `w-5` seems enough. (Can you review it just in case?

```suggestion
        <div className="w-5"><ChainLogo chainId={chain.id} /></div>
        <span className="ml-1 text-neutral-950">{chain.name}</span>
```